docs/walkthrough_transcribe.md

```diff
@@ -2,22 +2,115 @@
+# Walkthrough: transcribe with eScriptorium
 
-!!! warning
-    The section needs to be completed.
+Once the [segmentation of the images](walkthrough_segment.md) have been achieved and [some annotations](walkthrough_annotate.md) have been added, if needed, the next step in eScriptorium will be the text recognition of the documents and it is possible to do this step manually or automatically.
+
+## Automatic transcription
+
+### Obtaining a model
+
+You need a model to transcribe automatically, here are some ways to get one:  
+
+- by [training one with eScriptorium](walkthrough_train.md) or (via the command line) with [kraken](https://kraken.re),
+- by having one shared with you, either within [eScriptorium](walkthrough_collaborate.md#share-a-model), or sent to you as a file through other means of communication,
+- by importing a model from a file. Such files can be retrieved on the [Zenodo community](https://zenodo.org/communities/ocr_models/) dedicated to Kraken models.
+
+### Predicting the transcription
+
+To learn how to apply the text recognition model and obtain a prediction, see ["Predict a segmentation or a transcription"](walkthrough_predict.md).  
 
 ## Manual transcription
 
-## Automatic transcription
+To work on the transcription manually, you need to be on the "Edit" tab of your document, which is available either by clicking on the "Edit" button once you are in your document or in a square when you hover upon the thumbnail of an image once your are in the following URL: `{base_url}/document/{document-id}/images/`.
+
+Out of the five panels available in eScriptorium, two allow you to create and edit the transcription: "Transcription" and "Text".
+
+### The "Transcription" panel
+
+The "Transcription" panel displays the text in a way that imitates the layout of the document.  
+
+![image: Transcription panel on eScriptorium (access button framed in red)](img/transcribe/transcription_panel.png "Transcription panel on eScriptorium (access button framed in red)")  
+
+To start transcribing, click on the line you want to edit (it will be highlighted in the "Image" panel). An input window is then displayed.  
+
+Pressing ++enter++ or ++arrow-down++ will record the text and display the input field for the next segment. You can also use ++arrow-up++ to go back to the previous segment.  
+
+![image: Illustration of the transcription edit in the Transcription panel.](img/transcribe/by_hand.gif "Illustration of the transcription edit in the Transcription panel")
+
+You can also modify an existing transcription. The technique is the same as for the manual transcription: you click on the line, modify what is necessary and then press ++enter++. Each modification will be save in a history listing the user or model responsible for the editions.
+
+![image: Correction of the transcription in the Transcription panel.](img/transcribe/correction.gif "Correction of the transcription in the Transcription panel")
+
+### The "Text" panel
```

Comment:
I think you can insist more on line re-ordering in that section

Reply:
I think line re-ordering falls under segmentation and transcription, we can repeat the information in each walkthrough. I think the feature should be explicit since you are presenting the text panel :thinking:

Reply:
Okay !!! I will add the feature in my presentation and a gif of demonstration !!!